action.yaml

```diff
@@ -0,0 +1,17 @@
+name: Environment Variables Validate
```

Comment:
This file should be on the folder of the python script under a folder called examples

Reply:
placing the `action.yaml` file in an examples folder may be misleading since typically the `action.yaml` file is a critical part of the Action's functionality, not just an example. @hugoalmeida264